components/script/dom/htmlanchorelement.rs

```diff
@@ -581,21 +582,39 @@ pub fn follow_hyperlink(subject: &Element, hyperlink_suffix: Option<String>) {
     let source = document.browsing_context().unwrap();
 
     // Step 4-5: target attribute.
-    let target_attribute_value = subject.get_attribute(&ns!(), &local_name!("target"));
-
+    let subject_type = subject.get_attribute(&ns!(), &local_name!("target"));
```

Comment:
My preference would be to extract this into a separate function that has a comment linking to https://html.spec.whatwg.org/multipage/semantics.html#get-an-element's-target and invoke that, so that this code more clearly follows the specification.